ref/gl/gl_opengl.c

```diff
@@ -585,6 +589,9 @@ qboolean GL_CheckExtension( const char *name, const dllfunc_t *funcs, const char
 				if(( f = gEngfuncs.GL_GetProcAddress( name )))
 				{
 					// GL_GetProcAddress prints errors about missing functions, so tell user that we found it with different name
+#ifdef XASH_GLES
+					if(i != 0)
```

Comment:
This is so ugly. I don't know what to do about this, but this better be refactored somehow.